inflammation/models.py

```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
```

Comment:
There is already a function to do calculate the mean (`daily_mean`) so should call this function instead of calculating the mean within this standard deviation function

Reply:
You are correct, there is indeed a function that already calculates the mean. Therefore, I shall follow your advice and use that function to calculate the mean